llvm/test/CodeGen/AMDGPU/eliminate-frame-index-s-mov-b32.mir

```diff
@@ -368,9 +368,9 @@ body:   |
     ; GFX8-NEXT: S_NOP 0, implicit-def $sgpr56_sgpr57_sgpr58_sgpr59_sgpr60_sgpr61_sgpr62_sgpr63
     ; GFX8-NEXT: V_CMP_EQ_U32_e32 0, killed $vgpr0, implicit-def $vcc, implicit $exec
     ; GFX8-NEXT: S_CMP_EQ_I32 $sgpr4, $sgpr5, implicit-def $scc
+    ; GFX8-NEXT: $vgpr0 = V_MOV_B32_e32 64, implicit $exec
```

Comment:
This test also should get gfx10, gfx11, and gfx12 runlines (maybe leave for a separate commit though)

Reply:
added.